cpg_workflows/jobs/exomiser.py

```diff
@@ -79,6 +74,7 @@ def family_vcf_from_gvcf(family_members: list[SequencingGroup], out_path: str) -
     # -0 to replace missing with WT (potentially inaccurate, but predictable parsing in exomiser)
     # --threads 4 to use 4 threads
     # -Oz to write a compressed VCF
+    # type: ignore
```

Comment:
Is this suppressing some mypy nonsense? I thought `# type: ignore` was to ignore mypy errors on a specific line

Reply:
It's confusing mypy nonsense that I'm only getting locally, the CI passes here... I'm removing this line